src/components/flag/flag.jsx

```diff
@@ -53,17 +54,17 @@ Flag.defaultProps = {
   round: false,
 };
 
-const unNest = (acc, i) => acc.concat(i);
-const dupInUpperCase = arr => arr.map(code => [code, code.toUpperCase()]).reduce(unNest, []);
-
-const countryCodes = dupInUpperCase(['ca', 'de', 'fr', 'ru', 'gb', 'dk', 'fi', 'no', 'se', 'us', 'jp', 'cn', 'eu']);
-const currencyCountryCodes = dupInUpperCase(['dk', 'no', 'se', 'us', 'eu']);
-
 Flag.propTypes = {
   className: PropTypes.string,
   /** A valid 2-character country code */
-  countryCode: PropTypes.oneOf(countryCodes),
-  secondaryCountryCode: PropTypes.oneOf(currencyCountryCodes),
+  countryCode: PropTypes.oneOf([
```

Comment:
it's an issue even further upstream. https://github.com/reactjs/react-docgen/issues/36